client/pom.xml

```diff
@@ -61,7 +61,6 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
-      <scope>test</scope>
```

Comment:
it does not mean introducing new deps, because celeborn-common already has this dependency,